addons/stock/models/product.py

```diff
@@ -1100,3 +1100,23 @@ def _adjust_uom_quantities(self, qty, quant_uom):
         else:
             computed_qty = self._compute_quantity(qty, procurement_uom, rounding_method='HALF-UP')
         return (computed_qty, procurement_uom)
+
+
+class ProductSupplierInfo(models.Model):
+    _inherit = 'product.supplierinfo'
+
+    delay = fields.Integer(inverse='_inverse_delay')
+
+    def _inverse_delay(self):
+        if self.product_id:
+            orderpoint_domain = []
+            buy_route = self.env.ref('purchase_stock.route_warehouse0_buy', raise_if_not_found=False)
```

Comment:
Seems messy to reference a record from another module that extends off of this one :thinking: why not put this override in `purchase_stock`?